tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "allowJs": true,
     "downlevelIteration": true,
     "lib": ["es2017", "dom"],
+    "module": "es2020",
```

Comment:
Lazy loading requires `import()` syntax to work. This was discussed and approved in proposal.